packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarTextButtonIOS.ios.js

```diff
@@ -91,7 +93,9 @@ const BpkNavigationBarTextButtonIOS = (props: Props) => {
   const accessibilityTraits = ['button'];
   if (disabled) {
     accessibilityTraits.push('disabled');
-    tintColorFinal = disabledTintColor || colorGray300;
+    const gray300 =
+      theme && theme.colorGray300 ? theme.colorGray300 : colorGray300;
```

Comment:
this pattern seems to be coming up a lot. Maybe we should have a helper that can be used along the lines of `gray100ForTheme(theme)`, `gray300ForTheme(theme)` etc

Reply:
Probably should live in `utils.js` of `bpk-theme`?